src/router/middlewares/setContentType.test.ts

```diff
@@ -33,7 +33,7 @@ describe('setContentTypeMiddleware', () => {
         const nextMock = jest.fn();
```

Comment:
Update test description to 'request should return application/json if user request application/json in accept header'